validation/security-configs_test.go

```diff
@@ -17,29 +17,42 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-var checkResolutions = func(t *testing.T, r standard.Resolutions, chainID uint64, client *ethclient.Client) {
-	for contract, methodToOutput := range r {
+func resolveAddress(t *testing.T, chainID uint64, contract string) (Address, error) {
```

Comment:
Nit: could we change this function name to `getAddressFromConfig` and change the argument `contract` --> `contractName`